code/src/cljs/sixsq/nuvla/ui/pages/deployment_sets_detail/events.cljs

```diff
@@ -94,6 +95,27 @@
           [:dispatch [::main-events/action-interval-delete
                       {:id refresh-action-depl-set-id}]]]}))
 
+
+(defn untitled-deployments [resources]
+  (->> resources
+       (filter (fn [depl-group]
+                 (re-matches #"Untitled deployment group \d+" (:name depl-group))))))
+
+(reg-event-fx
+  ::set-default-name
+  (fn [{{:keys [::deployments-spec/deployment-sets ::spec/default-name-set?] :as db} :db}]
+    (when-not default-name-set?
+      (let [no-untitled-deployments (count (untitled-deployments (:resources deployment-sets)))]
```

Comment:
risk to fail if user create to untitled-documents and this dep-group is moved to another page. New untitled will start at 1 and a collision of name can happen.

Isn't it better to create a name with a date 'Deployment group 
2024-04-11T15:07:06.020Z'